### PR TITLE
Update DLsite.com.xml

### PR DESCRIPTION
**Pull request by imba-tjd**
Add more domains and remove `mixedcontent`(Didn't see anything broken during my limited usage, though I can't 100% guarantee).

Note two of them are `img.dlsite.jp` and `dlsite.blogimg.jp` which are not the subdomain of `dlsite.com` (especially `dlsite.blogimg.jp`, it's used in `b.dlsite.net`), but they are indeed used by DLsite. I don't know whether it's appropriate. If not, should I create another `.xml` entry?

And two of them will redir to http once. I have sent a feedback email, ~~thus I marked this PR as WIP.~~

---

* `https://trial.dlsite.com` will redir to `http://trial.origin.dlsite.com/error404.html` but the actual trial links work (for example `https://trial.dlsite.com/doujin/RJ119000/RJ118525_trial.zip`).
* `https://dealer.dlsite.com` will redir to `http://dealer.dlsite.com/login/...` but `https://dealer.dlsite.com/login` works.

From the email they said it's on their TODO list, but the priority seems not high.

### CODE REVIEW

**Comment by imba-tjd** [comment]
I think this is ready for reviews.

**Comment by ivysrono** [comment]
emm, this site is blocked in mainland……

**Comment by imba-tjd** [comment]
That's exactly the reason of this PR = =

**Comment by ivysrono** [comment]
有没有登录墙？

**Comment by imba-tjd** [comment]
网站有的功能是需要登录的，但是访问根目录都可以用https，除了主楼那俩

**Comment by ivysrono** [comment]
按照我所了解的本项目风格，建议这样处理：
主楼内容写进注释：
那俩添加 test，zip文件太大可能造成失败，如果找不到图片就保留在注释中；
相关又不同的域名新建 .xml，在注释开头写明关联。
争取这周能审，反正你也等了两周了不在乎再等两天吧😃

**Comment by imba-tjd** [comment]
`trial` is a domain for downloading, I don't think there is a picture in it.

`dealer` is added to test.

Created two other `.xml`. I wonder whether you mean adding the comment to `dlsite.com.xml` or the new `.xml`? Now I only added a `Rules related to this` to `dlsite.com.xml`.

File: src/chrome/content/rules/DLsite.com.xml [hunk 1]
```diff
@@ -1,44 +1,74 @@
 <!--
+	Related rules:
+	- dlsite.blogimg.jp
```
Comment (ivysrono) [inline src/chrome/content/rules/DLsite.com.xml]: .xml

File: src/chrome/content/rules/DLsite.jp.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!--
+	Related rules:
+	- dlsite.com
```
Comment (ivysrono) [inline src/chrome/content/rules/DLsite.jp.xml]: .xml

File: src/chrome/content/rules/blogimg.jp.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<!--
+	Related rules:
+	- dlsite.com
```
Comment (ivysrono) [inline src/chrome/content/rules/blogimg.jp.xml]: .xml

File: src/chrome/content/rules/DLsite.com.xml (hunk 1 above)
Comment (imba-tjd) [reply src/chrome/content/rules/DLsite.com.xml]: `dlsite.blogimg.jp.xml` or `blogimg.jp.xml` or change that filename to `dlsite.blogimg.jp.xml`?

File: src/chrome/content/rules/DLsite.jp.xml (hunk 2 above)
Comment (imba-tjd) [inline src/chrome/content/rules/DLsite.jp.xml]:
```suggestion
	- DLsite.com.xml
```

File: src/chrome/content/rules/blogimg.jp.xml (hunk 3 above)
Comment (imba-tjd) [inline src/chrome/content/rules/blogimg.jp.xml]:
```suggestion
	- DLsite.com.xml
```

File: src/chrome/content/rules/DLsite.com.xml [hunk 4]
```diff
@@ -1,44 +1,74 @@
...
+	- img.dlsite.jp
```
Comment (imba-tjd) [inline src/chrome/content/rules/DLsite.com.xml]:
```suggestion
	- DLsite.jp.xml
```

File: src/chrome/content/rules/DLsite.com.xml (hunk 1 above)
Comment (imba-tjd) [inline src/chrome/content/rules/DLsite.com.xml]:
```suggestion
	- blogimg.jp.xml
```

**Comment by imba-tjd** [comment]
……为什么不squash，这样很乱 = =

**Comment by ivysrono** [comment]
额，之前几个都是squash，为啥这个又变成了默认选项……
没事，这个项目管理一般般，不在乎这些小事😭

**Pull request merged**